node/actors/bft/src/leader/replica_commit.rs

```diff
@@ -105,7 +105,9 @@ impl StateMachine {
         // ----------- All checks finished. Now we process the message. --------------
 
         // Add the message to the QC.
-        commit_qc.add(&signed_message, self.config.genesis());
```

Comment:
add a comment that this should always succeed, because the check was already performed.